gpu-service/jobs.py

```
@@ -51,25 +60,53 @@
-except Exception:
-    logger.info("Creating Qdrant collection 'faces'")
-    qdrant.create_collection(
-        collection_name="faces",
-        vectors_config=models.VectorParams(size=128, distance=models.Distance.COSINE)
-    )
+def load_or_create_deeplake_dataset(path: str, creds: dict):
+    """Open DeepLake dataset; create if it does not exist."""
+    try:
+        ds_local = deeplake.open(path, creds=creds)
+        logger.info("Loaded DeepLake dataset at %s", path)
+        return ds_local
```

Ruff (0.14.7)

Consider moving this statement to an `else` block

(TRY300)